.github/workflows/publish.yml

```diff
@@ -13,53 +17,136 @@ on:
         required: false
         default: ''
         type: string
-      pure_python_wheel:
-        description: Whether to build a pure Python wheel - if not, separate wheels are built for each platform.
+      upload_to_pypi:
+        description: Always upload to PyPI - if not, only upload tags
         required: false
         default: false
         type: boolean
+      repository_url:
+        description: The PyPI repository URL to use
+        required: false
+        default: ''
+        type: string
+      submodules:
+        description: Whether to checkout submodules
+        required: false
+        default: true
+        type: boolean
+    secrets:
+      pypi_password:
+        required: false
 
 jobs:
 
+  targets:
+    name: Load build targets
+    runs-on: ubuntu-latest
+    outputs:
+      sdist: ${{ steps.set-outputs.outputs.sdist }}
+      universal: ${{ steps.set-outputs.outputs.universal }}
```

Comment:
We should avoid the term universal which was a mistake in the original templates (universal means pure Python but also compatible with Python 2 and 3). Maybe we can call this pure_python instead?

Reply:
Okay, I'll change that back.